src/include/sof/trace/preproc.h

```diff
@@ -62,9 +62,9 @@
 #define META_IF(condition) _META_IIF(META_BOOL(condition))
 
 /* primitive recursion
- * default depth is 1024
+ * default depth is 8
  */
-#define META_RECURSE(...) _META_REQRS_1024(__VA_ARGS__)
+#define META_RECURSE(...) _META_REQRS_8(__VA_ARGS__)
```

Comment:
I guess we could then remove `_META_REQRS_1024()..._META_REQRS_16()` but that doesn't have to go with this PR. Looks like `_META_DEFER_N()` and `META_DEFER()` are unused too , maybe some other "magic" macros too.

Reply:
>I guess we could then remove _META_REQRS_1024()..._META_REQRS_16() but that doesn't have to go with this PR.

I add it to commit with `META_RECURSE` depth reduction, because it's a point from which those are unused.